RELEASE/scripts/autoscend.ash

```diff
@@ -1073,9 +1079,8 @@ boolean doVacation()
 	}
 	if(in_zelda())
 	{
-		autoEquip($item[work boots]);
-		//maximizer is unreliable at that location for some reason.
-		equip($item[work boots]);
+		zelda_equipTool($stat[moxie]);
+		equipMaximizedGear();
```

Comment:
I'm not sure it's worth doing a full equipment-maximize before entering the shore. It's not like it matters, except for equipping work boots, which we were already doing. No big deal either way.

Reply:
Does zelda equip tool actually immediately wear the tool or does it just add it to maximizer? I thought it was only doing the latter. so without equipMaximizedGear it will not actually wear the items.

Reply:
I just tested it. 
`zelda_equipTool($stat[moxie]);`
does NOT equip the tool immediately. it just adds it to maximizer. without the second line it does not work.
```
> call scripts\tt_test.ash

[INFO] - Equipping fancy boots to slot acc1
Preference _auto_maximize_equip_acc1 changed from to fancy boots
```
checking my actual character he is not actually wearing the boots. meaning this is necessary

Reply:
split off into its own PR #342